docs/metrics.md

```diff
@@ -73,6 +73,10 @@ The `armory.metrics` module contains functionality to measure a variety of metri
 We have implemented the metrics in numpy, instead of using framework-specific metrics, to prevent expanding the required set of dependencies.
 Please see the relevant submodules in [armory/metrics](../armory/metrics/) for more detailed descriptions.
 
+### Custom Metrics
```

Comment:
we have a custom metrics section [here](https://github.com/twosixlabs/armory/blob/53708efa13f9582f317a4358215cd7a368c8c874/docs/metrics.md#custom-metrics-1). As far as I can tell, this TODO should be deleted

Reply:
You are right, I'm not sure why my rebase didn't apply properly, I am going through to see where it diverged now.

Reply:
Addressed in f07ae034e7a4415bf0bfeb3c543133bdf4e7b194.